src/game/scenes/mainmenu/menu_main.c

```diff
@@ -105,9 +105,11 @@ component *menu_main_create(scene *s) {
     menu_attach(menu, textbutton_create(&tconf, "ONE PLAYER GAME", NULL, COM_ENABLED, mainmenu_1v1, s));
     menu_attach(menu, textbutton_create(&tconf, "TWO PLAYER GAME", NULL, COM_ENABLED, mainmenu_1v2, s));
     menu_attach(menu, textbutton_create(&tconf, "TOURNAMENT PLAY", NULL, COM_ENABLED, mainmenu_mechlab, s));
+#ifdef DEBUGMODE
     component *net = textbutton_create(&tconf, "NETWORK PLAY", NULL, COM_ENABLED, mainmenu_enter_network, s);
     widget_set_id(net, NETWORK_BUTTON_ID);
     menu_attach(menu, net);
+#endif
```

Comment:
Maybe worth putting an `#else` with a disabled "NETWORK PLAY" button, similar to the "HELP" menu we haven't implemented yet?

Reply:
Added the disabled network play item.